data/human/hails.txt

```diff
@@ -3204,13 +3211,14 @@ phrase "friendly militia 4 variation"
 	word
 		" what "
 	word
-		"pirates"
-		"criminals"
-		"warlords"
-		"marauders"
-		"rampant piracy"
+		"pirates have"
+		"criminals have"
+		"warlords haves"
```

Comment:
_SCREEEEEEEEEEEE_

Reply:
I'm not having another PR take away from this one so easily!